docs/graphql/manual/guides/docker-networking.rst

```diff
@@ -0,0 +1,66 @@
+.. meta::
+   :description: Docker networking with Hasura
+   :keywords: hasura, docs, deployment, network, docker
+
+.. _docker_networking:
+
+Docker networking 
+=================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 2
+  :local:
+
+Introduction
+------------
+
+Sometimes you might want to connect Hasura with APIs that are either running outside of Docker or in a different Docker container.
+Depending on the setting, the network config is different. This section shows how to connect in each of these use cases.
+
+Network config
+--------------
+
+.. list-table:: 
+   :stub-columns: 1
+   :header-rows: 1
+
+   * - 
+     - Mac
+     - Windows
+     - Linux
+     - Comment
+   * - **Hasura to API (outside Docker)**
+     - ``host.docker.internal:3000``
+     - ``docker.for.win.localhost:3000``
+     - ``localhost:3000`` with (``--net=host``)
```

Comment:
We need to account for the case without `--net=host` as that will be more common esp if Hasura was launched via docker-compose.

E.g. `172.17.0.1:3000` (see docker bridge ip via `ifconfig`)